Description: Patch silences GCC warnings for overflow on implicit constant
    conversions.
Author: FreeBSD BLT maintainers
Last-Modified: Tue, 08 Jul 2014 12:19:13 +0400

--- a/generic/bltHash.h.in
+++ b/generic/bltHash.h.in
@@ -88,7 +88,7 @@
  * Acceptable key types for hash tables:
  */
 #define BLT_STRING_KEYS		0
-#define BLT_ONE_WORD_KEYS	((size_t)-1)
+#define BLT_ONE_WORD_KEYS	-1
 
 /*
  * Forward declaration of Blt_HashTable.  Needed by some C++ compilers
